nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -776,6 +776,7 @@ In addition to numerous new and upgraded packages, this release has the followin
   sudo mkdir /var/lib/redis-peertube
   sudo cp /var/lib/redis/dump.rdb /var/lib/redis-peertube/dump.rdb
   ```
+- Added the `keter` NixOS module. Keter is an apploader, it Binds to the main port (usually port 80) and reverse proxies requests to your application based on virtual hostnames. It provides graceful redeployment support, by launching a second copy of your application, performing a health check, and then switching reverse proxying to the new process. 
```

Comment:
Look around at [all of these](https://github.com/NixOS/nixpkgs/blob/ad8276cd21c9ef01e07e299ab20337722db5195a/nixos/doc/manual/release-notes/rl-2205.section.md#new-services-sec-release-2205-new-services). They're short, try to make your sentence smaller. (Drop the details)

Reply:
I dropped the details